### DISCUSSION TITLE
Türkiye IBAX üyeleri

### DISCUSSION THREAD

**Discussion by Oktayy72** [IBAX-io/go-ibax#1613] [Q&A]
Türkiye'den Katılan kullanıcılar çok fazla IBAX ı seviyorlar. Türkiye'ye özel olarak bir ödül verilmesi gerekiyor benim düşüncem.

**Comment by OnurPeker38** [comment]
kesinlikle bu kadar türk kullanıcının ilgisi karşılıksız kalmamalı

**Comment by beyuzman** [comment]
katılıyorum
